src/components/global/Playground/index.tsx

```diff
@@ -89,6 +93,10 @@ export default function Playground({
     }
   }, [isDarkTheme]);
 
+  useEffect(() => {
+    import('./device-preview.js').then((comp) => comp.defineCustomElement());
```

Comment:
Any reason for doing a dynamic import as opposed to the `import { }` syntax?

Reply:
Honestly, I cannot recall. I know I saw this usage pattern _somewhere_ when researching custom elements, but it doesn't appear to have a different to the functionality here. I'll swap that and then merge.

Reply:
Now I remember... haha SSR `HTMLElement` doesn't play nicely if not a dynamic import. Moving it back.

Reply:
Are we able to do this instead? https://github.com/ionic-team/ionic-framework/blob/main/packages/react/src/utils/HTMLElementSSR.ts

Ionic react uses this to get the HTMLElement class to work in SSR